lib/rules/require-scoped.ts

```diff
@@ -23,21 +26,174 @@ module.exports = {
+                                  },
+                                  fix(fixer: RuleFixer) {
+                                      return fixer.replaceText(
+                                          forbiddenAttr,
+                                          singleAllow,
+                                      )
+                                  },
+                              }
+                            : {
+                                  messageId: "remove",
+                                  data: {
+                                      attribute: forbiddenAttrName,
+                                  },
+                                  fix(fixer: RuleFixer) {
+                                      return fixer.remove(forbiddenAttr)
+                                  },
+                              },
+                    ],
+                })
+            }
+
+            /* eslint-disable no-inner-declarations -- conditional report function */
+            /**
+             * Reports the given node.
+             * @param {ASTNode} node node to report
+             */
+            function reportForbiddenPlain(node: AST.VElement) {
+                const singleAllow = allows.length === 1 && allows[0]
+
+                reporter.report({
+                    node: node.startTag,
+                    messageId: "forbiddenPlain",
+                    data: {
+                        attribute: allows
+                            .map((allow) => `\`${allow}\``)
+                            .join(", "),
+                    },
+                    suggest: singleAllow
+                        ? [
+                              {
+                                  messageId: "add",
+                                  data: {
+                                      attribute: singleAllow,
+                                  },
+                                  fix(fixer: RuleFixer) {
+                                      const close = tokenStore.getLastToken(
+                                          node.startTag,
+                                      )
+                                      return (
+                                          close &&
+                                          fixer.insertTextBefore(
+                                              close,
+                                              ` ${singleAllow}`,
+                                          )
+                                      )
+                                  },
+                              },
+                          ]
+                        : undefined,
+                })
+            }
+            /* eslint-enable no-inner-declarations -- conditional report function */
+
+            return {
+                "Program:exit"() {
+                    for (const style of styles) {
+                        if (style.scoped && style.module) {
+                            // report error can't have both
```

Comment:
Wanted to report an error here that says "Can't have both `scoped` and `module` on a single style`, but maybe that's out of scope for this rule. But on the other hand, it's kind of excessive to make a new rule just for that.

What do you think?